### ISSUE TITLE
Accessibility - RTL - The button should be on the left side and move when opening the block drawer

### ISSUE THREAD

**Issue by acquaalta** [ponlawat-w/moodle-local_accessibility#28]
Moodle version: 4.5
Plugin version: 2.2.0

Hi,
In Moodle in **LTR** mode with the plugin active and the block drawer **closed**, the screen looks like this:

![Image](https://github.com/user-attachments/assets/475f5da1-2979-484b-a81f-3e0cfacbaa35)

In Moodle in **LTR** mode with the plugin active and the block drawer **opened**, the screen looks like this:

![Image](https://github.com/user-attachments/assets/11151ae8-f6e3-461a-b8c2-f358a111a8ed)

The behavior of the location of the plugin icon looks correct in LTR mode. However, things go wrong in RTL mode. 

In Moodle in **RTL** mode with the plugin active and the block drawer **closed**, the screen looks like this:

![Image](https://github.com/user-attachments/assets/9a9b3f20-4f24-48c0-87f6-5d4dc185b9b2)

In Moodle in **RTL** mode with the plugin active and the block drawer **opened**, the screen looks like this:

![Image](https://github.com/user-attachments/assets/bb88eab4-15b8-4540-b5fa-d7673fe817bc)

**Comment by acquaalta** [comment]
Hi @adamjenkins and @ponlawat-w,
Is there any chance that you'll fix this issue soon and release a new version of the plugin?
We really like the plugin , but this is a blocker issue for us.

Thanks even just for considering it 🙏

**Comment by ponlawat-w** [comment]
Hi @acquaalta, this issue has been fixed with version 2.2.1 and released in this GitHub repository. For upgrading from Moodle Plugin Database, it will be published there soon. Thank you.

**Comment by acquaalta** [comment]
> Hi [@acquaalta](https://github.com/acquaalta), this issue has been fixed with version 2.2.1 and released in this GitHub repository. For upgrading from Moodle Plugin Database, it will be published there soon. Thank you.

Hi @ponlawat-w,
(1) Thank you very much for fixing the issue 😃 
(2) Is it possible that the publishing process of the plugin got stuck on Moodle end? In the Moodle Plugins website, the latest version of the plugin is still 2.2.0, although approximately 3 weeks passed.

Thanks

**Comment by adamjenkins** [comment]
Hi @acquaalta ,

Apologies for the hold up in publishing to the plugins website. We're at the start of the academic year here and so I simply got overwhelmed with other duties. Also, I would like to release this as 5.0 compatible so have been testing to make sure that is true. I will try to push it through to the plugins directory ASAP.
